server/src/test/java/com/ca/lsp/cobol/service/delegates/outline/OutlineTaskTest.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright (c) 2020 Broadcom.
+ * The term "Broadcom" refers to Broadcom Inc. and/or its subsidiaries.
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *   Broadcom, Inc. - initial API and implementation
+ */
+
+package com.ca.lsp.cobol.service.delegates.outline;
+
+import com.ca.lsp.cobol.service.MyDocumentModel;
+import com.ca.lsp.cobol.service.delegates.validations.AnalysisResult;
+import com.ca.lsp.core.cobol.semantics.outline.OutlineTree;
+import lombok.Value;
+import org.eclipse.lsp4j.DocumentSymbol;
+import org.eclipse.lsp4j.SymbolInformation;
+import org.eclipse.lsp4j.jsonrpc.messages.Either;
+import org.junit.Test;
+
+import java.util.Collections;
+import java.util.List;
+
+import static org.junit.Assert.assertEquals;
+
+public class OutlineTaskTest {
+  @Test
+  public void collectSymbols() {
+    DocumentSymbol theDocumentSymbol = new DocumentSymbol();
+    MyDocumentModel documentModel = new MyDocumentModel("", AnalysisResult.builder().outlineTree(
+        new OutlineTree(List.of(theDocumentSymbol), true)).build());
+    List<Either<SymbolInformation, DocumentSymbol>> expectedResult = List.of(Either.forRight(theDocumentSymbol));
+    assertEquals(expectedResult, OutlineTask.collectSymbols(documentModel));
+  }
+
+  @Test
+  public void collectSymbolsForNullDocumentModel() {
+    assertEquals(List.of(), OutlineTask.collectSymbols(null));
+  }
+
+  @Test
+  public void collectSymbolsForNullOutlineTree() {
+    MyDocumentModel documentModel = new MyDocumentModel("", AnalysisResult.builder().build());
+    assertEquals(List.of(), OutlineTask.collectSymbols(documentModel));
+  }
+
+  @Value
+  private class Cursor {
+    int line;
+    int character;
+  }
```

Comment:
looks like this class is not used

Reply:
Oops. You are right. I deleted the whole test to be sure =)